TagsterWebAPI/TagsterWebAPI.UnitTests/TagsControllerTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using TagsterWebAPI.Controllers;
+using Xunit;
+
+namespace TagsterWebAPI.UnitTests
+{
+   public class TagsControllerTests
+    {
+        [Fact]
+        public void InsertTagsData()
```

Comment:
There is nothing checking. If class is unnecessary, delete it

Reply:
This cannot be found after sync. I assume it should be resolved now